sql-plugin/src/main/303/scala/com/nvidia/spark/rapids/shims/spark303/Spark303Shims.scala

```diff
@@ -1,42 +0,0 @@
-/*
- * Copyright (c) 2021-2022, NVIDIA CORPORATION.
```

Comment:
why was this one removed and not moved?  m

Reply:
Because I am reusing the SparkShimImpl from the folder `301until304`